### PR TITLE
added autocomplete and seed word parsing

### PR DESCRIPTION
**Pull request by Sm00g15**
I can pretty easily add this to the RestoreBackup component if/when necessary as well.

### CODE REVIEW

**Comment by Sm00g15** [comment]
this also strips out any " " and , so that folks can just copy/paste directly from a Tezos Faucet json file

**Comment by developer0623** [comment]
error

**Comment by developer0623** [comment]
@Sm00g15 When I copy 15 words at a time, it

File: app/containers/HomeAddAddress/index.js [hunk 1]
```diff
@@ -237,24 +240,46 @@ class AddAddress extends Component<Props> {
...
+  seedPhraseConvert = (inputValue) => {
+    const words = inputValue.replace(/["\s]/g, "")
+    const seedString = words.replace(/,/g, " ")
+    const seedWords = seedString.split(/\s+/)
+    this.setState({ seeds: seedWords })
+  }
+
+  onChangeInput = (val) => {
+    if(val.length > 15) {
+      this.seedPhraseConvert(val)
```
Comment (developer0623) [inline app/containers/HomeAddAddress/index.js]: The seedPharseConvert function has to a return value as the Array type.
Please remove `return`

**Comment by umurb** [comment]
- [ ] Placeholder (floatingLabel) is missing. It should say `15 Word Secret Key`

screen shot 2018-11-15 at 12 18 27 pm

 - [ ] Show warning message, change the background color and disable the Import button if something's wrong with the seed phrase.

**Warning message styling should match the other warning messages. E.g:**
screen shot 2018-11-15 at 12 31 11 pm

**If a word is invalid, highlight the word with a different background color. E.g:**
valid - invlid
Colors used in above example; EBB5B5 & F6D6D6

### Cases
**User enters more than 15 words**
Warning Message: `Seed phrases with more than 15 words are not accepted`

screen shot 2018-11-15 at 12 22 49 pm

**User enters an invalid word**
Warning Message: `Invalid word. Please double check Xth word`

screen shot 2018-11-15 at 12 25 38 pm


Full list of valid words can be found at: `app/components/RestoreBackup/seed.json`

**Comment by Sm00g15** [comment]
just a heads up, after coming back to this since my last commit I'm hitting this error whenever running yarn dev in the Tezori folder: 

(nothing has been changed locally)

```
Uncaught Error: package.json does not exist at /package.json
    at Object.exports.find (pre-binding.js:18)
    at Object.eval (usb.js:3)
    at eval (usb.js:526)
    at Object../node_modules/usb/usb.js (renderer.dev.dll.js:95921)
    at __webpack_require__ (renderer.dev.dll.js:63)
    at eval (listenDevices.js:19)
    at Object../node_modules/@ledgerhq/hw-transport-node-hid/lib/listenDevices.js (renderer.dev.dll.js:220)
    at __webpack_require__ (renderer.dev.dll.js:63)
    at eval (TransportNodeHid.js:55)
    at Object../node_modules/@ledgerhq/hw-transport-node-hid/lib/TransportNodeHid.js (renderer.dev.dll.js:194)
```

The issue is referenced here: https://github.com/grpc/grpc/issues/13049#issuecomment-337677610 ; 

This is also not an issue on the newly created branch that @umurb created from the Cryptonomic/develop branch for #476 . This issue seems like it could be solved by moving all the code to a new branch on develop like the merge of #476

**Comment by umurb** [comment]
Noted, closing this PR.

**Pull request closed without merging**